document/core/exec/instructions.rst

```diff
@@ -3077,66 +3077,69 @@ on the top of the stack.
 
  5. While the stack is not empty and the top of the stack is not an :ref:`exception handler <syntax-handler>`, do:
 
-    a. Pop the top element from the stack, prepending it to the :ref:`throw context <syntax-ctxt-throw>` of the exception: :math:`\XT[\val^n~(\THROWadm~a)]`.
+    a. Pop the top element from the stack.
 
  6. Assert: The stack is now either empty, or there is an exception handler on the top of the stack.
 
  7. If the stack is empty, then:
 
-    a. **TODO** *Return a result value representing the uncaught exception (will probably just be the same as 11.a.i. below).*
+    a. Return the uncaught exception :math:`\val^n~(\THROWadm~a)` as a value.
 
-.. todo::
-   After PR #221 is resolved, this step should be filled in with a PR to specify uncaught exception results.
+8. Else:
 
-8. Else there is an :ref:`exception handler <syntax-handler>` :math:`H` on the top of the stack.
+   a. Assert: there is an :ref:`exception handler <syntax-handler>` :math:`H` on the top of the stack.
 
 9. Pop the exception handler :math:`H` from the stack.
 
-10. Assert: :math:`H` is either of the form :math:`\CATCHadm\{a^?~\instr^\ast\}^k` or :math:`\DELEGATEadm\{l\}.`
+10. Assert: :math:`H` is either of the a catching or a delegating :ref:`exception handler <syntax-handler>`.
```

Comment:
Is this assertion needed? That appears to be true by definition of exception handler.

Reply:
Removing this line.